ts/store/reducers/persistedPreferences.ts

```diff
@@ -57,3 +67,6 @@ export default function preferencesReducer(
 
   return state;
 }
+
+export const isPagoPATestEnabledSelector = (state: PersistedPreferencesState) =>
+  state.isPagoPATestEnabled;
```

Comment:
@valentinacala you can check that `isPagoPATestEnabled === true` to cover the case when `isPagoPATestEnabled` is `undefined`